docs/reference/catalogbase-reference.md

```diff
@@ -41,67 +37,47 @@ catalog.get_collections()
 The `get_data_products()` function returns a list of data products.
 
 ```python
-get_data_products(
-    basic=True
-)
+get_data_products(basic)
```

Comment:
why remove `basic=True`?
I am guessing this is a typo? :)

Reply:
ah, I see. Because you want to be consistent everywhere by not initializing the variable?

But this just seems weird. It seems like we are passing `basic` as a variable.